src/engine/prague.md

```diff
@@ -9,19 +9,63 @@ This specification is based on and extends [Engine API - Cancun](./cancun.md) sp
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
+- [Structures](#structures)
+  - [ExecutionPayloadV4](#executionpayloadv4)
+  - [PayloadAttributesV4](#payloadattributesv4)
 - [Methods](#methods)
   - [engine_newPayloadV4](#engine_newpayloadv4)
     - [Request](#request)
     - [Response](#response)
     - [Specification](#specification)
-  - [engine_getPayloadV4](#engine_getpayloadv4)
+  - [engine_forkchoiceUpdatedV4](#engine_forkchoiceupdatedv4)
     - [Request](#request-1)
     - [Response](#response-1)
     - [Specification](#specification-1)
+  - [engine_getPayloadV4](#engine_getpayloadv4)
+    - [Request](#request-2)
+    - [Response](#response-2)
+    - [Specification](#specification-2)
   - [Update the methods of previous forks](#update-the-methods-of-previous-forks)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Structures
+
+### ExecutionPayloadV4
```

Comment:
I don’t think we need a new ExecutionPayload definition, since we have `targetBlobsPerBlock` passed as a param to `newPayloadV4`

Reply:
that's the value coming from the CL, the EL needs a way to capture what is on-chain

the EL's job here is to verify the value in the EL header matches what the CL is sending

Reply:
right, but the treatment of this value looks similar to `parentBeaconBlockRoot`, i don’t get why the `targetBlobsPerBlock` should be a part of the payload

Reply:
if we don't do it this way, we would need to have the CL validate this field in the `ExecutionPayload`

and I just took a look at how we handle the parent beacon block root, and kind of agree we should do this check there and can skip passing it via the `ExecutionPayloadV4` here

let's see if we decide to ship EIP-7742 in pectra and we can make these updates in that case